ethsigner/core/src/integration-test/java/tech/pegasys/ethsigner/jsonrpcproxy/model/jsonrpc/SendRawTransaction.java

```diff
@@ -49,16 +50,16 @@ public String request() {
 
   @SuppressWarnings("unchecked")
   public String request(final Request<?, EthSendTransaction> request, final long chainId) {
-    final List<Transaction> params = (List<Transaction>) request.getParams();
-    final Transaction transaction = params.get(0);
+    final List<JsonObject> params = (List<JsonObject>) request.getParams();
+    final JsonObject transaction = params.get(0);
```

Comment:
will this be a list of transactions or just a single transaction? Previous code was dealing with list of transactions and we were fetching the first one.

Reply:
never mind, I misread that.